redis-persistence/src/main/java/com/netflix/conductor/dao/dynomite/RedisExecutionDAO.java

```diff
@@ -257,6 +257,7 @@ public boolean exceedsInProgressLimit(Task task) {
 
 		long current = getInProgressTaskCount(task.getTaskDefName());
 		if(current >= limit) {
+			logger.info("Task execution count limited. task - {}:{}, limit: {}, current: {}", task.getTaskId(), task.getTaskDefName(), limit, getInProgressTaskCount(task.getTaskDefName()));
```

Comment:
getInProgressTaskCount method hits dynomite and also has side effect, use `current`.